sim/.scalafmt.conf

```diff
@@ -0,0 +1,269 @@
+    "glob:**src/main/scala/midasexamples/AutoCounterModule.scala",
+    "glob:**src/main/scala/midasexamples/AutoCounterValidator.scala",
+    "glob:**src/main/scala/midasexamples/Config.scala",
+    "glob:**src/main/scala/midasexamples/EnableShiftRegister.scala",
+    "glob:**src/main/scala/midasexamples/GCD.scala",
+    "glob:**src/main/scala/midasexamples/GlobalResetConditionTests.scala",
+    "glob:**src/main/scala/midasexamples/InstanceNameHelper.scala",
+    "glob:**src/main/scala/midasexamples/MultiReg.scala",
+    "glob:**src/main/scala/midasexamples/MultiRegfile.scala",
+    "glob:**src/main/scala/midasexamples/MultiSRAM.scala",
+    "glob:**src/main/scala/midasexamples/MulticlockAutoCounterModule.scala",
+    "glob:**src/main/scala/midasexamples/MulticlockPrintfModule.scala",
+    "glob:**src/main/scala/midasexamples/NestedModels.scala",
+    "glob:**src/main/scala/midasexamples/Parity.scala",
+    "glob:**src/main/scala/midasexamples/PassthroughModels.scala",
+    "glob:**src/main/scala/midasexamples/PointerChaser.scala",
+    "glob:**src/main/scala/midasexamples/PrintfModule.scala",
+    "glob:**src/main/scala/midasexamples/Regfile.scala",
+    "glob:**src/main/scala/midasexamples/ResetPulseBridgeTest.scala",
+    "glob:**src/main/scala/midasexamples/ResetShiftRegister.scala",
+    "glob:**src/main/scala/midasexamples/Risc.scala",
+    "glob:**src/main/scala/midasexamples/RiscSRAM.scala",
+    "glob:**src/main/scala/midasexamples/ShiftRegister.scala",
+    "glob:**src/main/scala/midasexamples/Stack.scala",
+    "glob:**src/main/scala/midasexamples/Termination.scala",
+    "glob:**src/main/scala/midasexamples/TriggerPredicatedPrintf.scala",
+    "glob:**src/main/scala/midasexamples/TriggerWiringModule.scala",
+    "glob:**src/main/scala/midasexamples/TrivialMulticlock.scala",
+    "glob:**src/main/scala/midasexamples/TwoAdders.scala",
+    "glob:**src/main/scala/midasexamples/Util.scala",
+    "glob:**src/main/scala/midasexamples/VerilogAccumulator.scala",
+    "glob:**src/main/scala/midasexamples/WireInterconnect.scala",
+    "glob:**src/test/scala/fasedtests/FASEDTestSuite.scala",
+    "glob:**src/test/scala/midasexamples/TutorialSuite.scala"
+  ]
+}
+
+maxColumn = 120
+align.preset = most
+continuationIndent.defnSite = 2
+assumeStandardLibraryStripMargin = true
+
+docstrings.style = SpaceAsterisk
+
+lineEndings = preserve
+optIn.configStyleArguments = true
+danglingParentheses.preset = true
+verticalMultiline.excludeDanglingParens = []
+trailingCommas = multiple
+
+align.tokens."+" = [
+  {
+    code = "=",
+    owners=[{regex="Term.Param|Defn|Term.Assign"}]
+  },
+  {
+    code = ":",
+    owners = [{regex="Term.Param|Decl.Def"}]
+  },
+  {
+    code = ":="
+  },
+  {
+    code = "<>"
+  },
+  {
+    code = ":*="
+  },
+  {
+    code = ":=*"
+  },
+  {
+    code = ":*=*"
+  },
+  {
+    code = ":<="
+  },
+    {
+    code = ":=>"
+  },
+  {
+    code = ":<>"
+  }
+]
+
+includeCurlyBraceInSelectChains = false
```

Comment:
I think this looks better enabled. https://scalameta.org/scalafmt/docs/configuration.html#includecurlybraceinselectchains

Reply:
I'm inclined to agree, so i'll switch it. It does differ from what FIRRTL uses, and i'm not sure what effect it'll have on chisel naming. Hopefully none.